plugins/processors/awsapplicationsignals/internal/normalizer/attributesnormalizer.go

```diff
@@ -9,47 +9,63 @@ import (
 
 	"go.opentelemetry.io/collector/pdata/pcommon"
 	conventions "go.opentelemetry.io/collector/semconv/v1.18.0"
+	semconv "go.opentelemetry.io/collector/semconv/v1.18.0"
 	"go.uber.org/zap"
 
-	attr "github.com/aws/amazon-cloudwatch-agent/plugins/processors/awsappsignals/internal/attributes"
+	"github.com/aws/amazon-cloudwatch-agent/plugins/processors/awsapplicationsignals/common"
+	attr "github.com/aws/amazon-cloudwatch-agent/plugins/processors/awsapplicationsignals/internal/attributes"
+)
+
+const (
+	MaxEnvironmentLength         = 259
+	MaxServiceNameLength         = 255
+	DefaultMetricAttributeLength = 1024
```

Comment:
could you help to add a comment to explain why these 3 metric attributes have different max length?

Reply:
Added.